src/actions/colorModes.js

```diff
@@ -0,0 +1,55 @@
+export function framingBackgroundColorFor(museumMode) {
+    return switchOn(
+        museumMode,
+        {backgroundColor: '#2D2D2D'},
+        {backgroundColor: '#eff2f5'}
+    )
+}
+
+export function contentBackgroundColorFor(museumMode) {
+    return switchOn(
+        museumMode,
+        {backgroundColor: '#333333'},
+        {backgroundColor: 'white'}
+    )
+}
+
+export function separatorColorFor(museumMode) {
+    return switchOn(
+        museumMode,
+        {borderBottomColor: 'black'},
+        {borderBottomColor: '$borderGrey'}
+    )
+}
+
+export function deselectedTabColorFor(museumMode) {
+    return switchOn(
+        museumMode,
+        {backgroundColor: '#282828'},
+        {backgroundColor: '$lightestGrey'}
+    )
+}
+
+export function helpTextColorFor(museumMode) {
+    return switchOn(
+        museumMode,
+        {color: '#addde0'},
+        {color: 'rgba(0,93,105,1)'}
+    )
+}
+
+export function textColorFor(museumMode) {
+    return switchOn(
+        museumMode,
+        '#addde0',
+        'black'
+    )
+}
+
+function switchOn(mode, withModeStyle, withoutModeStyle) {
+    if (mode) {
```

Comment:
It could indeed! And that's a common way to condense control flow, for sure.

The tradeoff with a ternary is that its brevity (and, in the case of JavaScript, its functionality) depends on executing a single operation—the return of the value—from each branch. So if I wanted to perform logging, printing, or debugging with a ternary conditional, I would first be forced to remove it from the ternary into a multi-line conditional like the one shown here.  This function's relatively broad usage footprint (it gets executed in a lot of different places) makes it a likely place for a developer to want to debug if they have questions about how our conditional styling works, or want to understand whether or not a method that _uses_ this method has the appropriate inputs if it's functioning in a way other than the user expects. This already happened to me a few times because I passed my dark mode and light mode arguments to client functions in the wrong order :). 

Because that's likely to happen a lot to this piece of code relative to others, it's important that I make this piece of code as easy to modify for debugging as possible. Leaving it as a multiline conditional facilitates that.

In fact, for me, the heuristic described above simplifies to the following rule of thumb: leave it multiline, rather than ternary, _unless_ a multiline statement obfuscates additional information that a developer would otherwise be able to collect from the _shape_ of the code they're looking at. So I'll use a ternary, say, in a variable assignment while writing JSX, because the shape of the JSX conveys a lot of meaning, and I want to keep my property assignments one-line if possible to convey that meaning successfully. 

Sorry for noveling :)